www/assets/js/databrowser.js

```diff
@@ -504,6 +504,7 @@ browser = simply.app({
             browser.actions.saveChangesSpreadsheet()
         },
         import: (el, value) => {
+            browser.actions.removeErrorsFromElement()
```

Comment:
as it is just one line of code, and only relevant here, creating a separate action for it seems overkill
please just inline the code